core/src/test/java/org/apache/spark/launcher/SparkLauncherSuite.java

```diff
@@ -155,6 +157,10 @@ public void testRedirectToLog() throws Exception {
 
   @Test
   public void testChildProcLauncher() throws Exception {
+    // This test is failed on Windows due to the failure of initiating executors
+    // by the path length limitation. See SPARK-18718.
+    assumeTrue(!Utils.isWindows());
```

Comment:
It seems there are not (at least I could not find) proper workarounds to deal with this problem on Windows rather than reducing the command line. I believe we don't want to introduce a lot of changes to pass the tests on Windows. So, I rather disabled this. Please let me know if there is any suggestion.